code/modules/client/preference_setup/loadout/items/uniform.dm

```diff
@@ -49,30 +49,6 @@
 	path = /obj/item/clothing/under/color/colorable
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
-/datum/gear/uniform/skirt
```

Comment:
This can't be removed. Removing this will break people's loadouts in ways more than one. Have to re-add a skirt, and any customization they had (name, color, desc, etc).

And if you want just a basic skirt+shirt combo... what are you supposed to do now? The loadout preview is going to look really bad, unless you get shorts or some other uniform piece under the skirt, cause as this is coded currently in the PR you can't have just a skirt. And on spawn, every time, you'd have to remove all your accessories, including the skirt (accessory), put it on, and put all the accessories back on the shirt (now a uniform)

I think it should be as it was originally - skirts being both in the uniform category and in the accessory category. It could be the same actual item, but they must exist in both categories, otherwise it is ass.